services/ext/api.go

```diff
@@ -1063,8 +1063,8 @@ func (api *PublicAPI) SyncDevices(ctx context.Context, name, picture string) err
 	return api.service.messenger.SyncDevices(ctx, name, picture, nil)
 }
 
-func (api *PublicAPI) EnableAndSyncInstallation(request *requests.EnableAndSyncInstallation) error {
-	return api.service.messenger.EnableAndSyncInstallation(request)
+func (api *PublicAPI) EnableInstallationAndSync(request *requests.EnableInstallationAndSync) (*protocol.MessengerResponse, error) {
```

Comment:
As I understand, this is the only breaking change in this PR.

If you really think we should rename it, then:
1. Mark `EnableAndSyncInstallation` as deprecated, but keep it working.
2. Introduce `EnableInstallationAndSync` and update clients to use the new endpoint. 

This way we can ensure clients are still compatible with the changes.

Reply:
I'm not fully aware of all the implications, but as I understood it, the breaking change it bigger than that. Once this is released, people will only be able to pair between 2.31 and up.

I guess it was not possible to keep backwards compatibility, otherwise Andrea would have thought of it.

As for renaming the API, both desktop and mobile are creating PRs at the same time, so I think it's fine to just rip the band aid right now and not keep an unused API?

Reply:
I think it depends on the terminology of a `breaking change` that we should agree upon.
Let me discuss this with the @status-im/status-go-guild today, I'll be back with an answer in the evening.

Reply:
We have 3 types of breaking change:

- Breaking between versions (the type that we want to avoid the most)
- Breaking of functionality (basically introducing a (or several) bug(s))
- Breaking between `status-go` and client (potentially this case)

I think breaks between `status-go` and clients can be managed, as long as they are managed they are only a problem if someone has an old branch and doesn't rebase against the latest `develop`.

Reply:
Indeed, this would be considered as breaking change, because we're breaking compatibility with previous versions 👍

Reply:
@qfrank I'm not really saying that we should stick to the old function name. 
But there's a way to make this carefully: create a new function and deprecate the old one. Then the switch to the new function name will be seamless 🙌

Reply:
Oh! I only read your comment and and didn't check the code 😄